app/assets/javascripts/components/overview/course_approval.jsx

```diff
@@ -0,0 +1,243 @@
+import React from 'react';
+
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import createReactClass from 'create-react-class';
+import Select from 'react-select';
+import selectStyles from '../../styles/single_select';
+import { fetchSpecialUsers } from '../../actions/settings_actions';
+import { fetchAllCampaigns, addCampaign } from '../../actions/campaign_actions';
+import { addUser } from '../../actions/user_actions';
+import { getCourseApprovalStaff } from '../../selectors';
+import { STAFF_ROLE } from '../../constants';
+
+
+const CourseApproval = createReactClass({
```

Comment:
I'd prefer new components to be functional components, if possible. That will make it easier to follow future developments in React, I think.

Reply:
Okay